modules/swagger-codegen/src/main/resources/php/README.mustache

```diff
@@ -6,8 +6,8 @@
 This PHP package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
 - API version: {{appVersion}}
-- Package version: {{artifactVersion}}
-- Build date: {{generatedDate}}
+{{#artifactVersion}}- Package version: {{artifactVersion}}
+{{/artifactVersion}}- Build date: {{generatedDate}}
```

Comment:
You can just put `{{#artifactVersion}}` and `{{/artifactVersion}}` on lines of their own.

As of recently whitespace will be stripped around them if they are the only thing on a line.

This looks nicer and more readable in the mustache template.